### DISCUSSION TITLE
How do I set pulsar-proxy High availability?

### DISCUSSION THREAD

**Discussion by HGHNice** [apache/pulsar#22118] [Q&A]
How do I set pulsar-proxy High availability in k8s

**Comment by eolivelli** [comment]
Pulsar proxy is a stateless service, like the brokers.
You can start as many instances as you will.

**Reply by HGHNice** [reply]
Thank you. Do you mean that I only need to modify the replicas of pulsar-proxy? This is Deployment k8s
![屏幕截图 2024-02-26 153029](https://github.com/apache/pulsar/assets/84365959/a5345b8a-d600-411c-97ed-ffb0ce6cec53)
